bundles/org.openhab.binding.govee/src/main/java/org/openhab/binding/govee/internal/GoveeHandler.java

```diff
@@ -318,12 +320,16 @@ public void updateDeviceState(@Nullable StatusResponse message) {
         if (newColorTempInKelvin != lastColorTempInKelvin) {
             logger.trace("Color-Temperature Status: old: {} K {}% vs new: {} K", lastColorTempInKelvin,
                     newColorTempInPercent, newColorTempInKelvin);
-            updateState(CHANNEL_COLOR_TEMPERATURE_ABS, new QuantityType<>(lastColorTempInKelvin, Units.KELVIN));
-            updateState(CHANNEL_COLOR_TEMPERATURE, new PercentType(newColorTempInPercent));
-        }
+            if (lastColorTempInKelvin >= 2000 && lastColorTempInKelvin <= 9000) {
```

Comment:
`newColorTenpInKelvin`

Besides that, wouldn't clamping the new color temperature/brightness be the better idea compared just preventing the state update? Say the update happened from 2000 to 9001 ... in that case I'd say a state of 9000 is more correct than a state of 2000.

Reply:
when coding this fix i decided to do no update when the value is invalid. Auto-correcting values is a slippery slope and can become inimitable to users.
This PR is just about reducing the impact on an external bug from Govee.

Reply:
So in my example you'd rather accept/expect a reading of 2000 instead of 9000? Or in the case of ramping up brightness, which is then reported as 101, rather keep the old value of say 1 instead of reporting 100?
Can't say I agree there, in Ecovacs binding at least I did the clamping to the valid range.